lipsweb/models.py

```diff
@@ -0,0 +1,18 @@
+from cms.models.pluginmodel import CMSPlugin
+
+from django.db import models
+
+
+class Submission(CMSPlugin):
+    content = models.TextField(default='')
+    upvoteCount = models.PositiveIntegerField(default=0)
+
+    def countSubmission(self):
```

Comment:
why do these functions exist? I can't see anywhere they are used and the second one seems to be wrong (can't take a len of an integer). Delete them?